Makefile

```diff
@@ -437,20 +437,9 @@ endif
 
 ifeq ($(COMPILER),ido)
 $(BUILD_DIR)/src/boot/driverominit.o: OPTFLAGS := -O2
```

Comment:
Why did you ultimately decide not to include driverominit?

Reply:
just being conservative, the only evidence we'd have is it needs -O2 which isn't enough for me
it would also mean mi0 would be libu64, of which there's 0 evidence besides being after driverominit

but idk